dev-python/argon2-cffi/argon2-cffi-21.2.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+inherit distutils-r1
+
+DESCRIPTION="CFFI bindings to the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2-cffi"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-python/argon2-cffi-bindings-21.2.0[${PYTHON_USEDEP}]"
+BDEPEND="
+	test? (
+		dev-python/hypothesis[${PYTHON_USEDEP}]
+	)
+"
+
+DOCS=( AUTHORS.rst CHANGELOG.md FAQ.rst README.rst )
+
+src_prepare() {
+	# Patch the tool.flit.module.name entry in pyproject.toml to include the "src" subfolder
```

Comment:
Please keep lines below 80 chars (or 72, if you prefer).

Reply:
(unless it's really impossible)

Reply:
Of course. I reformatted the code in `src_prepare` to stay below 80 chars. The exception is the second sed substitution which will always be exceed the character count with the proper indentation.

I also checked the second ebuild (argon2-cffi-bindings) and it looks fine with regards to line length.